src/xdocs/index.xml.vm

```diff
@@ -96,6 +96,15 @@
         For the exact day, please follow our
         <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
       </p>
+      <p>
+        Checkstyle is following Emotional Semantic Versioning:
```

Comment:
There is an established term for versioning like this: "romantic versioning". I suggest we use this (apparently well-known term) to lend some legitimacy to our system. See https://dafoster.net/articles/2015/03/14/semantic-versioning-vs-romantic-versioning/.

Reply:
done